monai/networks/utils.py

```diff
@@ -436,6 +439,58 @@ def copy_model_state(
     return dst_dict, updated_keys, unchanged_keys
 
 
+def get_state_dict(obj: Union[torch.nn.Module, Dict]):
+    """
+    Get the state dict of input object if has `state_dict`, otherwise, return object directly.
+    For data parallel model, automatically convert it to regular model first.
+
+    Args:
+        obj: input object to check and get the state_dict.
+
+    """
+    if isinstance(obj, (nn.DataParallel, nn.parallel.DistributedDataParallel)):
+        obj = obj.module
+    return obj.state_dict() if hasattr(obj, "state_dict") else obj  # type: ignore
```

Comment:
thanks! please help update this one: https://github.com/Project-MONAI/MONAI/blob/f08f1d3de838bd3674a3696387447624fcc0cf1f/monai/networks/utils.py#L406-L411

Reply:
Sure, updated in the latest commit.

Thanks.